README.md

```diff
@@ -1,8 +1,71 @@
-gatling-maven-plugin-demo-kotlin
-================================
+Gatling plugin for Maven demo project
+=====================================
 
-Simple showcase of a Kotlin maven project using the gatling-maven-plugin.
+A simple showcase of a Maven project using the
+[Gatling plugin for Maven](https://github.com/gatling/gatling-maven-plugin). The full documentation for the plugin can
+be found [on the Gatling website](https://gatling.io/docs/current/extensions/maven_plugin/).
```

Comment:
Please don't advertise the GitHub repository. We want users to be rooted to our website, not GitHub.